crates/orderbook/src/orderbook.rs

```diff
@@ -42,24 +47,18 @@ struct Metrics {
     orders: prometheus::IntCounterVec,
 }
 
+#[derive(Display)]
+#[strum(serialize_all = "snake_case")]
 enum OrderOperation {
     Created,
     Cancelled,
 }
 
-fn operation_label(op: &OrderOperation) -> &'static str {
-    match op {
-        OrderOperation::Created => "created",
-        OrderOperation::Cancelled => "cancelled",
-    }
-}
-
-fn order_class_label(class: &OrderClass) -> &'static str {
-    match class {
-        OrderClass::Market => "user",
-        OrderClass::Liquidity => "liquidity",
-        OrderClass::Limit => "limit",
-    }
+#[derive(Display)]
+#[strum(serialize_all = "snake_case")]
+enum OrderClass {
+    User,
```

Comment:
I think `user` was probably a misnomer to begin with. Let's call it `market` instead and adjust the grafana alerts accordingly.

Reply:
If we rename this, why don't we call it `Market` and `OutOfMarket` or somethinglike this? Technically a market order is also a limit order.

In fact, it could maybe simply become a boolean `inMarket`?

Reply:
Since we have everywhere "market" and "limit" still, I would suggest to stick with that, otherwise we add inconsistencies. We could clean up all the "market" and "limit" code, and then unify it properly (we also have the "liquidity" there...)